datahub-web-react/.eslintrc.js

```diff
@@ -19,29 +18,34 @@ module.exports = {
         project: './tsconfig.json',
     },
     rules: {
-        eqeqeq: ['error', 'always'],
-        'react/destructuring-assignment': 'off',
-        'no-console': 'off',
-        'no-debugger': 'warn',
-        'require-await': 'warn',
+        '@typescript-eslint/no-explicit-any': 'warn',
```

Comment:
I think my preference here would be to try to make this upgrade as transparent as possible, for now. While it's more ideal to fix a lot of these, doing this in stages where we upgrade with the warnings=off, and then can address one at a time. Helps keep the changeset a bit smaller.

Reply:
Sure, I'll swap all the warnings to off then. You mentioned you tried rebasing locally - will you be updating this MR with the resolved conflicts or should I tackle that instead?

Reply:
I guess more specifically if you could set the warnings to match what the eslintrc had before, that would be great 👍 Not sure if they all really need to be necessarily off if we had some to warn before, thinking to keep parity unless there's a reason we can't do that.

Reply:
I won't be able to directly edit this PR due to permissions, so if you can do that here that would be ideal.

Reply:
The good news is CI passed on my test branch, so I'm gonna plan to do a bit more testing on it today. Then once that's feeling good, we can figure out a time to merge. One thing about this kind of PR is that it can lead to some interruption on developers side so we may want to do a little announcement or something that folks need to re-yarn / upgrade node ideally post-merging this.

Reply:
But we can handle that post-merge of course